### ISSUE TITLE
named import vs default import ?

### ISSUE THREAD

**Issue by everythinginjs** [mathias-rosa/MyDadJoke#1]
[Here](https://github.com/mathias-rosa/MyDadJoke/blob/9aa3d4cdc5425b06f74a9854193dba8d108ac849/src/pages/SearchResult.tsx#L1) 

is there a difference to have
import Paper from '@mui/material/Paper'
vs
import { Paper } from '@mui/material'

**Comment by mathias-rosa** [comment]
Yes using defaults import avoid pulling unused modules so it's prefered to use defaults imports rather than named imports with Material UI.
